modules/ROOT/pages/asm-secret-group-creation-task.adoc

```diff
@@ -159,7 +159,11 @@ The TLS context you store in Secrets Manager can only be used either when you co
 * *Name* - Enter a name for your TLS context. 
 * *TLS Version* - Select the TLS version. 
 * *Target* - Select the target for the TLS context from the drop-down list. +
-** *Anypoint Security* - Validates the SSL handshake for Runtime Fabric. 
+** *Anypoint Security* - Validates the SSL handshake for Runtime Fabric. +
+If you use the *Anypoint Security* target, you can select keystores and truststores that are either PEM or non-PEM types. Supported types include: +
+* JKS
+* PKCS12
+* JCEKS 
 ** *Mule* - Uses the TLS context as the SSL validation for Mule 4 based API proxies. 
```

Comment:
I think you mean for JKS, PKCS12, and JCEKS to be indented under Anypoint Security. If so, change the * to *** for all three.